### PR TITLE
Add footerlink to each dialog's preference pane

### LINKED ISSUE #1131: Dialogs should have link to TWPREFS
To make WP:TWPREFS more discoverable, the Twinkle dialogs should have a morebits footerlink pointing to the relevant section (for that module) at TWPREFS.

### PR DESCRIPTION
**Pull request by Amorymeltzer**
Closes #1131

See also #1133

### CODE REVIEW

File: modules/twinklespeedy.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ Twinkle.speedy.initDialog = function twinklespeedyInitDialog(callbackfunc) {
 	dialog.setTitle('Choose criteria for speedy deletion');
 	dialog.setScriptName('Twinkle');
 	dialog.addFooterLink('Speedy deletion policy', 'WP:CSD');
+	Window.addFooterLink('CSD prefs', 'WP:TW/PREF#speedy');
```
Comment (siddharthvp) [inline modules/twinklespeedy.js:50]:
```suggestion
	dialog.addFooterLink('CSD prefs', 'WP:TW/PREF#speedy');
```

**Review by siddharthvp — COMMENTED** [review]
LGTM, except for the one bug.

File: modules/twinklespeedy.js (hunk 1 above)
Comment (Amorymeltzer) [reply modules/twinklespeedy.js]: Heh, good eye, thanks.

Comment (Amorymeltzer) [reply modules/twinklespeedy.js]: One more: 8345a9d

**Pull request merged**